components/property_access.rst

```diff
@@ -81,6 +81,20 @@ You can also use multi dimensional arrays::
     var_dump($propertyAccessor->getValue($persons, '[0][first_name]')); // 'Wouter'
     var_dump($propertyAccessor->getValue($persons, '[1][first_name]')); // 'Ryan'
 
+.. tip::
+
+    If the key of the array contains a dot ``.``, a left squared bracket ``[``
+    or a right squared bracket ``]``, it is possible to escape these characters
```

Comment:
I checked in code, and escaping right squared bracket is "not supported" /needed
```suggestion
    If the key of the array contains a dot ``.`` or a left squared bracket, it is possible to escape these characters
```

Reply:
Did the appropriate change 👍 thanks!